src/openloco/input.h

```diff
@@ -54,47 +54,47 @@ namespace openloco::input
     };
 
     void init();
-    void init_mouse();
-    bool has_flag(input_flags value);
-    void set_flag(input_flags value);
-    void reset_flag(input_flags value);
+    void initMouse();
+    bool hasFlag(input_flags value);
+    void setFlag(input_flags value);
+    void resetFlag(input_flags value);
     input_state state();
     void state(input_state);
 
     gfx::point_t getMouseLocation();
-    bool is_hovering(ui::WindowType);
-    bool is_hovering(ui::WindowType, ui::window_number);
-    bool is_hovering(ui::WindowType type, ui::window_number number, ui::widget_index widgetIndex);
-    ui::widget_index get_hovered_widget_index();
+    bool isHovering(ui::WindowType);
+    bool isHovering(ui::WindowType, ui::window_number);
+    bool isHovering(ui::WindowType type, ui::window_number number, ui::widget_index widgetIndex);
+    ui::widget_index getHoveredWidgetIndex();
 
-    bool is_dropdown_active(ui::WindowType type, ui::widget_index index);
+    bool isDropdownActive(ui::WindowType type, ui::widget_index index);
 
-    bool is_pressed(ui::WindowType type, ui::window_number number);
-    bool is_pressed(ui::WindowType type, ui::window_number number, ui::widget_index index);
-    ui::widget_index get_pressed_widget_index();
+    bool isPressed(ui::WindowType type, ui::window_number number);
+    bool isPressed(ui::WindowType type, ui::window_number number, ui::widget_index index);
+    ui::widget_index getPressedWidgetIndex();
 
     void updateCursorPosition();
 
-    bool is_tool_active(ui::WindowType, ui::window_number);
+    bool isToolActive(ui::WindowType, ui::window_number);
     bool toolSet(ui::window* w, int16_t widgetIndex, uint8_t tool);
-    void cancel_tool();
-    void cancel_tool(ui::WindowType, ui::window_number);
+    void cancelTool();
```

Comment:
In my PR I switched this around to toolCancel to follow toolSet

Reply:
Ok, I renamed it.